spec/unit/provider/package/powershell_spec.rb

```diff
@@ -19,7 +19,7 @@
 require "spec_helper"
 require "chef/mixin/powershell_out"
 
-describe Chef::Provider::Package::Powershell, :windows_only do
+describe Chef::Provider::Package::Powershell, :windows_only, :windows_gte_10 do
```

Comment:
if these aren't working on older PS I think we have a bug. This should work pretty far back

Reply:
those were busted just like the powershell_exec ones

Reply:
https://buildkite.com/chef/chef-chef-master-omnibus-adhoc/builds/310 sending a new build through to verify

Reply:
that blows up on all the 2012 testers with " Minimum installed PowerShell Version required is 5"

Reply:
note this is the powershell package provider not the powershell script provider